src/core/api_client_parallel.py

```
@@ -1,116 +1,146 @@
 import time
-from typing import Dict, List, Optional, Any, Tuple
-from collections import defaultdict
-import json
-
 
+import aiohttp
 class ParallelAPIClient:
     """Parallel API client that uses multiple keys concurrently for maximum speed"""
-    
+
     def __init__(self, base_url: str, api_keys: List[str], requests_per_minute_per_key: int = 60):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)